instrumentation/aws-sdk/aws-sdk-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awssdk/v2_2/AwsSdkInstrumentationModule.java

```diff
@@ -30,7 +30,8 @@ public AwsSdkInstrumentationModule() {
       "io.opentelemetry.instrumentation.awssdk.v2_2.RequestType",
       "io.opentelemetry.instrumentation.awssdk.v2_2.SdkRequestDecorator",
       "io.opentelemetry.instrumentation.awssdk.v2_2.DbRequestDecorator",
-      "io.opentelemetry.instrumentation.awssdk.v2_2.TracingExecutionInterceptor"
+      "io.opentelemetry.instrumentation.awssdk.v2_2.TracingExecutionInterceptor",
+      "io.opentelemetry.extension.trace.propagation.AwsXRayPropagator"
```

Comment:
@iNikem Here

Reply:
I still don't understand :( Here you _allow_ it to be used, by injecting it into user's classloader. Where or how do you actually use it?

Reply:
Ah thought you remember from your PR comment before :P We use it here

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/instrumentation/aws-sdk/aws-sdk-2.2/library/src/main/java/io/opentelemetry/instrumentation/awssdk/v2_2/TracingExecutionInterceptor.java#L112

But it currently fails with `ClassNotFoundException` because I forgot to add the helper. Would be caught in tests after #1643.